invenio_files_rest/helpers.py

```diff
@@ -166,11 +176,17 @@ def send_stream(stream, filename, size, mtime, mimetype=None, restricted=True,
     # Set cache-control
     if not restricted:
         rv.cache_control.public = True
-        cache_timeout = current_app.get_send_file_max_age(filename)
+        cache_timeout = (
+            current_app.get_send_file_max_age(filename)
+            or current_app.config["FILES_REST_SEND_FILE_MAX_AGE"]
+        )
         if cache_timeout is not None:
             rv.cache_control.max_age = cache_timeout
             rv.expires = int(time() + cache_timeout)
 
+    if restricted:
+        rv.cache_control.no_cache = True
```

Comment:
@slint can you please check if this change and the one above (the new var `current_app.config["FILES_REST_SEND_FILE_MAX_AGE"]`) are ok with you?

Reply:
- Regarding, `FILES_REST_SEND_FILE_MAX_AGE`, can't we just document/comment that setting  [`SEND_FILE_MAX_AGE_DEFAULT`](https://flask.palletsprojects.com/en/2.1.x/config/#SEND_FILE_MAX_AGE_DEFAULT), which controlls `current_app.get_send_file_max_age(...)` should be enough?
- For the restricted files, I agree with the `no-cache`

Reply:
We could set the default in the cookiecutter `invenio.cfg` too. For any instances made with older cookiecutters they might still have an undefined max_age though.  WDTY? @ntarocco

Reply:
It should be ok because `SEND_FILE_MAX_AGE_DEFAULT` is part of Flask framework, added long time ago...

Reply:
So we should not set it in `invenio.cfg` ?